### PR TITLE
Notification Channel Setup

### PR DESCRIPTION
**Pull request by saish24**
Created 2 notification channels to notify user whenever a patient is admitted to the hospital.
Modified Calculate age function.

### CODE REVIEW

File: app/src/main/java/com/example/healthlog/HealthLog.java [hunk 1]
```diff
@@ -19,8 +25,36 @@ public void onCreate() {
...
+    private void createNotificationChannels() {
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+
+//          For high priority Notifications, eg new patient admitted etc.
+            NotificationChannel channel1 = new NotificationChannel(
+                    CHANNEL_1_ID,
+                    "Channel 1",
+                    NotificationManager.IMPORTANCE_HIGH
+            );
+            channel1.setDescription("This is channel 1");
```
Comment (iamsh4shank) [inline app/src/main/java/com/example/healthlog/HealthLog.java]: can you use string resources instead of hardcoding it :)

**Review by iamsh4shank — COMMENTED** [review]
Hey good job, can you put a screenshot here :) and also please squash the commits :)

**Comment by saish24** [comment]
**_Sample Screenshots_**
Screenshot_2020-12-18-03-42-39-60_5ee831692cd1396117f9de0ee465d043
Screenshot_2020-12-18-03-42-30-36_5ee831692cd1396117f9de0ee465d043

**Review by iamsh4shank — COMMENTED** [review]
Nice work :) please squash your commits :)

**Comment by iamsh4shank** [comment]
Hey @saish24 can you please squash your commits :)

**Pull request merged**